_includes/post-preview.html

```diff
@@ -0,0 +1,31 @@
+{% assign post = include.post %}
+
+{%- capture excerpt_link_url -%}
+  {%- if post.excerpt_link_url -%}
+    {{ post.excerpt_link_url }}
+  {%- else -%}
+    {{ site.baseurl }}{{ post.url }}
+  {%- endif -%}
+{%- endcapture -%}
```

Comment:
**issue**: Which is the most common and can we use that as the default?

**For example**
Pseudo code example, but set the fallback and use it as the default if nothing is passed.

```liquid
{% comment %} Hope there's a better way of combining these two vars {% endcomment %}
assign fallback_url = site.baseurl | append(post.url) ;

assign excerpt_link_url = post.excerpt_link_url | default(fallback_url)
```

Reply:
Good call. Updated in [c4d016a](https://github.com/uswds/uswds-site/pull/2979/commits/c4d016a972835cafeb014ae58c443dc913828ca4)